### ISSUE TITLE
CIVIC color theme fails foreground/background contrast accessibility requirements for text and borders

### ISSUE THREAD

**Issue by bradfordjanice** [hackoregon/civic#509]
## Description

While working on the button component I found that the red color of the border and text against the white background did not pass accessibility requirements.  Here is the message:

Element has insufficient color contrast of 4.17 (foreground color: #dc4556, background color: #ffffff, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1

### Steps to reproduce

You can see this message in Storybook, in the addon panel of the button component.  Look under the Accessibility tab.

### Expected result

Modify this color to meet the contrast requirements, the new color must fit in with the color theme.  

While working on this, it would be good to evaluate the entire color theme to see if there are any other places where accessibility requirements aren't met.  Knowledge of accessibility requirements and colors is valuable.

**Comment by jaronheard** [comment]
The red color (#dc4556) should be updated to CIVIC blue (#1E62BD)

**Comment by jaronheard** [comment]
#559 resolved

**Comment by fstorr** [comment]
Some colors still need addressing. I'd recommend using [The Paciello Group's Colour Contrast Analyser](https://developer.paciellogroup.com/resources/contrastanalyser/). Note: if you want to paste in a color value into that tool, you need to right-click onto a text field to get the Paste menu option.

Note that color contrast requirements for borders ([WCAG 2.1's non-text contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)) are different from [contrast requirements for text and images of text](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html).
